.env

```diff
@@ -21,6 +21,10 @@ REACT_APP_JLAB_FALLBACK_DOMAIN="water_224.h5"
 # Rendering performance profiling (local development only)
 REACT_APP_PROFILING_ENABLED=false
 
+# Disable ESLint error modal in development (log warnings instead)
+ESLINT_NO_DEV_ERRORS=true
```

Comment:
Yay! No more bloody error modal for an unused var!! Note that TS errors will still bring up a modal.

Reply:
Well, bad example, since we turn unused vars errors into warnings in our ESLint config, but you know what I mean.